app/src/components/main/analytics/events/ga4Events/apiDocumentationPageEvents.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { normalizeEventString } from '../../utils';
+
+const getBasicEventConfig = (place) => ({ place, action: 'click', category: 'api_documentation' });
+
+export const API_DOCUMENTATION_PAGE_EVENTS = {
+  CLICK_OPTION_BLOCK: ({ place, type, condition }) => ({
+    ...getBasicEventConfig(place),
+    type: normalizeEventString(type),
+    element_name: 'arrow_expand',
+    condition: normalizeEventString(condition),
+  }),
+
+  CLICK_CHANGE_DOCUMENTATION_TYPE: (place) => ({
+    ...getBasicEventConfig(place),
+    element_name: `button_${place}`,
```

Comment:
Yes, because GA4 need it to works correctly. 
![screenshot_09](https://user-images.githubusercontent.com/84175555/200759709-7013f7dd-dc84-4593-8815-fb135d34656c.jpg)

Reply:
Please contact the task creator and ask to update the document or or notify qa team because now this field does not exist